src/client/Microsoft.Identity.Client/AppConfig/AuthorityInfo.cs

```diff
@@ -461,7 +461,9 @@ public static IAuthorityValidator CreateAuthorityValidator(AuthorityInfo authori
             ///
             /// - if the authority is not defined at the application level and the request level is not AAD, use the request authority
             /// - if the authority is defined at app level, and the request level authority of is of different type, throw an exception
```

Comment:
needs some rewording here?

Reply:
Reword how? We throw if app and request authorities are of different types (AAD, B2C, ADFS) or have different hosts/clouds.

Reply:
I think he means grammatical error: remove "of" after the authority.